wallet/udb/txdb.go

```diff
@@ -900,6 +911,14 @@ func fetchRawCreditScriptLength(v []byte) uint32 {
 	return byteOrder.Uint32(v[86:90])
 }
 
+// fetchRawCreditScriptType returns the script type of the credit's pkScript
+func fetchRawCreditScriptType(v []byte) scriptType {
```

Comment:
sidenote: seems this is only being used to create an updated value during the upgrade, but is otherwise unused.  We should look into removing this in a later DB upgrade